src/database/models/Profile.ts

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable no-trailing-spaces */
+/* eslint-disable lines-between-class-members */
+/* eslint-disable indent */
+/* eslint-disable require-jsdoc */
+import { Model, DataTypes } from 'sequelize';
+import { sequelize } from '../config/sequelize.config';
+
```

Comment:
Because you are working on the profile user so you will use User model i think this profile model is not neccessary